extensions/resource-deployment/src/services/platformService.ts

```diff
@@ -36,6 +38,7 @@ export interface IPlatformService {
 	runCommand(command: string, options?: CommandOptions): Promise<string>;
 	saveTextFile(content: string, path: string): Promise<void>;
 	deleteFile(path: string, ignoreError?: boolean): Promise<void>;
+	backgroundExecuteNotebook(taskName: string | undefined, notebookInfo: string | NotebookInfo, notebookService: INotebookService, tempNoteBookPrefix: string): void;
```

Comment:
hmm, this should be in the notebook service.

Reply:
moved